.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!--
+NOTE:
+
+    If you're asking about how to use OpenSSL, this isn't the right 
+    forum.  Please see our
+    [User Support resources](https://github.com/openssl/openssl/blob/master/.github/SUPPORT.md)
+
```

Comment:
Side note: Should we include a note about reporting the version of OpenSSL if relevant in this generic template as well?

Reply:
Hmmm, maybe.  Can't hurt

Reply:
Done